components/Sidebar/SubheaderTab.tsx

```diff
@@ -0,0 +1,88 @@
+import ChevronRightIcon from '@mui/icons-material/ChevronRight'
+import {
+  Accordion,
+  AccordionSummary,
+  Typography,
+  AccordionDetails,
+  List,
+  ListItemButton,
+} from '@mui/material'
+import { useRouter } from 'next/router'
+import React, { Fragment, useState } from 'react'
+import { SubheaderData } from './sidebar_data'
+
+const SubheaderTab = ({
+  subheaderData,
+  tabSelected,
+  setTabSelected,
+  parentRoute,
+}: {
+  subheaderData: SubheaderData
+  tabSelected: string
+  setTabSelected: Function
+  parentRoute: string
+}) => {
+  const [expanded, setExpanded] = useState<boolean>(false)
+  const hasTabs: boolean = subheaderData.tabs.length > 0
+  const router = useRouter()
+
+  return (
+    <Fragment>
+      <Accordion
+        expanded={expanded}
+        onChange={() => {
+          if (!expanded && tabSelected != subheaderData.title) {
```

Comment:
use `!==` instead of `!=`. there are also multiple places where this was not followed can change them too